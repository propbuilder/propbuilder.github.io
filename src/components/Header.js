const Header = () => {
    return (
        <div className="header">
            <h3 className="widget-header">Stock Widget</h3>
        </div>
    );
};

export default Header;