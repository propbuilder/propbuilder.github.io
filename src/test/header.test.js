import '@testing-library/jest-dom/extend-expect';
import Header from "../components/Header";
import {
    cleanup,
    render
  } from '@testing-library/react';

  describe('Header component test', () => {
    afterEach(cleanup);
      test('1. Header component should render correctly', () => {
          const { container } = render(
              <Header />
          );
          expect(container).toMatchSnapshot();
      });
  });
