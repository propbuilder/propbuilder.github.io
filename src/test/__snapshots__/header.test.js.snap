// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component test 1. Header component should render correctly 1`] = `
<div>
  <div
    class="header"
  >
    <h3
      class="widget-header"
    >
      Stock Widget
    </h3>
  </div>
</div>
`;
